### PR TITLE
[FIX] Added Bio Structure for UserCard, rendering Skeleton View on loading Instead of [Object][Object] 

### LINKED ISSUE #20283: [BUG] Improper rendering of bio in UserCard
### Description:

When you click on a user, while the user card is loading,` MarkdownText.js` renders [ Object Object ] in place of bio ( after a few sec then correct bio shows up ).
Screenshot from 2021-01-20 17-07-56

### Steps to reproduce:

1. 
2. 
3. 

### Expected behavior:
It should just show a skeleton view.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.0-develop
- Operating System: Ubuntu
- Deployment Method:
- Number of Running Instances:
- DB Replicaset Oplog:
- NodeJS Version: 12.18,4
- MongoDB Version:4.2.8

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by yash-rajpal**
- [x] I have read the Contributing Guide 
  - [x] I have signed the CLA 
  - [x] Lint and unit tests pass locally with my changes
  - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [ ] I have added necessary documentation (if applicable)
  - [ ] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)

Added Bio Structure for rendering Skeleton View on loading UserCard.

## Issue(s)

fix #20283 
## Steps to test or reproduce

Check the Issue
## Further comments

Until UserCard received bio, it was passing Skeleton as content to Markdown, so it was returning [Object][Object], which was shown to user. So Made one more entity called as bioStructure, now if bio is present, bio will be passed to markdown and everything would work as it is supposed to, but if bio is not present, bioStructure would be rendered as skeletonView.

### CODE REVIEW

**Comment by yash-rajpal** [comment]
I hope the change is visible in GIF.
bio-structure-after

**Comment by pratyaksh123** [comment]
You could also do this way :+1: 
`{ bio && (
			<Info withTruncatedText={false} style={clampStyle} height='x60'>
				{typeof bio === 'string' ? <MarkdownText content={bio} /> : bio}
			</Info>
		)}`

File: client/components/UserCard.js [hunk 1]
```diff
@@ -81,7 +82,7 @@ const UserCard = forwardRef(({
 		{ customStatus && <Info>{customStatus}</Info> }
 		<Roles>{roles}</Roles>
 		<Info>{localTime}</Info>
-		{ bio && <Info withTruncatedText={false} style={clampStyle} height='x60'><MarkdownText content={bio}/></Info> }
+		{bio !== undefined ? bio && <Info withTruncatedText={false} style={clampStyle} height='x60'><MarkdownText content={bio}/></Info> : <Info withTruncatedText={false} style={clampStyle} height='x60'>{bioStructure}</Info>}
```
Comment (MartinSchoeler) [inline client/components/UserCard.js:85]: Maybe instead of doing this ternary, we can check the type of the bio variable inside of the info element?

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
Requested some changes, pratyaksh123 have some neat ideas to handling this. Thanks!

**Comment by yash-rajpal** [comment]
@MartinSchoeler Thanks, will do changes asap.

**Review by MartinSchoeler — APPROVED** [review]
Looks good to me!

**Comment by yash-rajpal** [comment]
@ggazzo Please review. :)

**Comment by MartinSchoeler** [comment]
@yash-rajpal please be patient, we are currently on a _feature-freeze_ so pull requests that are not directly related to the ones done in the current release, can only be merged on the next release (don't worry we are almost at the end of the _featrue-freeze_).

Mentioning other members of the team will not make it go faster 🙂 

Thanks!

**Pull request merged**